spec/integration/lib/auth0/auth0_client_spec.rb

```diff
@@ -42,4 +42,28 @@
     let(:credentials) { v2_credentials.merge({access_token: ENV["MASTER_JWT"]}) }
   end
 
+  context "client headers" do
```

Comment:
I think it will better to test this in an unit tests but we can have this here till we re-do them